Evaluation/workflow.R

```diff
@@ -0,0 +1,160 @@
+#' @title createQualificationReport

```

Comment:
workflow.R here is almost 1:1 copy of the template provided with the ospsuite.reportingengine package.
The only thing I needed to do: replace "qualification-plan.json" with "evaluation-plan.json"